From 8581342a3675a62458727fa4063d1dca1c538fdd Mon Sep 17 00:00:00 2001
From: Mikhail Shilkov <github@mikhail.io>
Date: Fri, 15 Sep 2023 17:59:18 +0200
Subject: [PATCH] Avoid panic on ResourceServer preview

---
 internal/auth0/resourceserver/resource.go | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/internal/auth0/resourceserver/resource.go b/internal/auth0/resourceserver/resource.go
index b4e797a1..43e85e3c 100644
--- a/internal/auth0/resourceserver/resource.go
+++ b/internal/auth0/resourceserver/resource.go
@@ -296,5 +296,6 @@ func isManagementAPI(state cty.Value) bool {
 		return false
 	}
 
-	return state.GetAttr("name").AsString() == auth0ManagementAPI
+	name := state.GetAttr("name")
+	return !name.IsNull() && name.AsString() == auth0ManagementAPI
 }
-- 
2.39.2 (Apple Git-143)

