From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Thomas Kappler <tkappler@pulumi.com>
Date: Fri, 29 Sep 2023 07:52:53 +0200
Subject: [PATCH] Patch to avoid upstream ResourceServer panic: code moved


diff --git a/internal/auth0/resourceserver/expand.go b/internal/auth0/resourceserver/expand.go
index be0a7002..8d4b7c8f 100644
--- a/internal/auth0/resourceserver/expand.go
+++ b/internal/auth0/resourceserver/expand.go
@@ -55,5 +55,6 @@ func resourceServerIsAuth0ManagementAPI(state cty.Value) bool {
 		return false
 	}
 
-	return state.GetAttr("name").AsString() == auth0ManagementAPIName
+	name := state.GetAttr("name")
+	return !name.IsNull() && name.AsString() == auth0ManagementAPIName
 }
